src/style.css

```diff
@@ -1 +1,53 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo__img {
+  display: block;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item:not(:last-child) {
+  padding-right: 20px;
```

Comment:
Here is not an inner distance, it's outer distance between elements. So, use margin